### DISCUSSION TITLE
Don't open every PR on every repo imaginable. It's annoying!

### DISCUSSION THREAD

**Discussion by KSXGitHub** [renovatebot/renovate#16266] [Suggest an Idea]
I have lost count of the number of repos I made, but most of them are inconsequential. So imagine my dismay when waking up to 100+ notifications all made by the Renovate Bot.

When I really need to use Renovate, I would have already created a `renovate.json` file. I don't need to be reminded all the time. It's annoying!

**Comment by rarkins** [comment]
I have disabled the bot for all of your repos.
